### ISSUE TITLE
Version 2.0: npm start fails for Windows

### ISSUE THREAD

**Issue by jgadsden** [OWASP/threat-dragon#355]
**Describe the bug**
After installing Threat Dragon using `pnpm install` , when running start command `npm start` the commands fail. This only happens on Windows platforms 

**Expected behaviour**
Command `npm start`  should start front-end and server not only for Linux/MacOS but also for Windows 

**Environment**
 - Version: 2.0
 - Platform:(Web App
 - OS: Windows
 - Browser: any of chrome, firefox, safari, etc

**To Reproduce**
1. `pnpm install`
2. `npm start`

**Any additional context, screenshots, etc**
```

> threat-dragon-server@1.5.8 start:dev
> pm2 --name td.server start 'npm run dev'

[PM2][ERROR] Script not found: td.server\'cd
ERROR: "start:server" exited with 1.
```

**Comment by jgadsden** [comment]
This bug was first reported by @taoyiyang1

**Comment by jgadsden** [comment]
This has a documentation fix in PR #354
